packages/dropdowns.next/src/views/combobox/StyledTrigger.ts

```diff
@@ -138,7 +138,7 @@ export const StyledTrigger = styled.div.attrs({
   'data-garden-id': COMPONENT_ID,
   'data-garden-version': PACKAGE_VERSION
 })<IStyledTriggerProps>`
-  overflow-y: auto;
+  overflow-y: ${props => (props.isBare ? 'visible' : 'auto')};
```

Comment:
Unfortunately, this is problematic for `isMultiselectable`

<img width="152" alt="Screenshot 2024-07-25 at 10 09 49 AM" src="https://github.com/user-attachments/assets/dc924290-0fa2-40ac-81b1-eb3e58c765a9">

Reply:
https://github.com/zendeskgarden/react-components/pull/1872/commits/97ac681e1d72e41ceebf43b24277f7802a60abd8 maintains the `overflow-y: auto` when `isMultiSelectable == true` 

![Screenshot 2024-07-26 at 8 39 42 AM](https://github.com/user-attachments/assets/a9bd94ca-3a19-4873-addd-c9672f043408)
![Screenshot 2024-07-26 at 8 39 56 AM](https://github.com/user-attachments/assets/0319a941-51f5-4b0c-8c57-b4b64ed1be4d)